contrib_bots/lib/ConverterBot/docs.md

```diff
@@ -0,0 +1,69 @@
+# Converter bot
+
+This bot allows users to perform conversions for various measurement units.
+
+## Usage
+
+Run this bot as described in [here](http://zulip.readthedocs.io/en/latest/bots-guide.html#how-to-deploy-a-bot).
+
+Use this bot with the following command
+
+`@convert <number> <unit_from> <unit_to>`
+
+This will convert `number`, given in the unit `unit_from`, to the unit `unit_to`
+and print the result.
+
+* `number` can be any floating-point number, e.g. 12, 13.05, 0.002.
+* `unit_from` and `unit_to` are two units from [the following](#supported-units) table.
+* `unit_from` and `unit_to` can be preceded by [these](#supported-prefixes) prefixes.
+
+### Supported units
+
+| Physical quantity | units |
+| ----------------- | ----- |
+| Information | Bit, Byte |
```

Comment:
Yep, we had a discussion on bits and bytes abbreviations on Zulip. Basically, the official aliases are "b" for bit and "B" for Byte - since the bot converts all input to lowercase, it couldn't differentiate between those units.